src/main/webapp/resources/js/pages/remote-apis/RemoteApiPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
File comment

Reply:
Added file comments in ee3e6c9714570d0c2f64f201b3dbef09c9f9d351